scripting/shavit-zones.sp

```diff
@@ -545,14 +554,23 @@ public void SQL_RefreshZones_Callback(Database db, DBResultSet results, const ch
 
 			iFreestyleRow++;
 		}
-
 		else
 		{
 			if(view_as<int>(type) >= MAX_ZONES || view_as<int>(type) < 0)
 			{
 				continue;
 			}
 
+			if(type == Zone_CustomSpawn)
+			{
+				gF_CustomSpawn[0] = results.FetchFloat(1);
+				gF_CustomSpawn[1] = results.FetchFloat(2);
+				gF_CustomSpawn[2] = results.FetchFloat(3);
+				gB_CustomSpawn = true;
```

Comment:
```
public void ClearCustomSpawn()
{
	for(int i = 0; i < 3; i++)
	{
		gF_CustomSpawn[i] = 0.0;
	}
	gB_CustomSpawn = false;
}
```

Line 744 - 751. 
This function get's called in UnloadZones and the SQL_DeleteCustom_Spawn_Callback callback.
Thought it would be more efficient only having one for loop written instead of 3